src/components/Book.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Kindly ensure you do not disable linters as they help us write cleaner code.